de-lib/src/main/java/org/iplantc/de/client/models/UserSettings.java

```diff
@@ -271,6 +272,20 @@ public Splittable asSplittable() {
         return ret;
     }
 
+    public void useDefaultValues(UserInfo instance) {
+        JSONObject defaults = new JSONObject();
+        JSONObject defaultHomeDir = new JSONObject();
+
+        JSONString id = new JSONString(instance.getHomePath());
+        JSONString path = new JSONString(instance.getHomePath());
```

Comment:
For this PR, I think it's a good idea to use `getHomePath` here.

If the bootstrap endpoint doesn't return the user's real home path because `data-info` is down, then I think this `getHomePath` method can return `/iplant/home/<username>` for now (it can be updated in another PR).

That static part of the path (`/iplant/home/`) should be configurable if it isn't already.